boards/arm/96b_carbon/doc/96b_carbon.rst

```diff
@@ -281,6 +310,50 @@ terminal:
 
    Hello World! arm
 
+.. _96b_carbon_verify_bluetooth:
+
+Verifying Bluetooth Functionality
+---------------------------------
+
+This section contains instructions for verifying basic Bluetooth
+functionality on the board. For help on Zephyr applications
+in general, see :ref:`build_an_application`.
+
+1. Flash the nRF51 with the hci_spi sample application as described in
+   :ref:`96b_carbon_nrf51_bluetooth`.
+
+2. Install the dfu-util flashing app, as described above.
+
+3. Build the ``samples/bluetooth/ipsp`` application for 96b_carbon::
```

Comment:
I would put the list of the obscure commands in the IPSP sample README, and link from here to it in a separate section of the 96b_carbon.rst, not the one that describes how to quickly check if the Bluetooth controller is up and running